heron/common/src/cpp/basics/mempool.cpp

```diff
@@ -21,5 +21,5 @@
 
 // TODO(nlu): get the pool size limit from config
 MemPool<google::protobuf::Message>* __global_protobuf_pool__ =
-                                   new MemPool<google::protobuf::Message>(50 * 1024 * 1024);
+                                   new MemPool<google::protobuf::Message>(512);
```

Comment:
Got it. Looks like your changing the 512 to be configurable. Shouldn't the config-driven value be used here now?